### PR TITLE
Adding my folder index file

### PR DESCRIPTION
**Pull request by wejrowski**
Check it out! let me know what you think everyone.

### CODE REVIEW

File: brent/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```
Comment (wejrowski) [inline brent/index.html:1]: Hey this is a great doctype tag! 😸

File: brent/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+<html>
```
Comment (wejrowski) [inline brent/index.html:2]: Nice html tag

File: brent/index.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    <head>
+    </head>
+    <body>
+        Hello world, this is brent.
```
Comment (adbruner) [inline brent/index.html:6]: The world says hi back :100:

File: brent/index.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
```
Comment (vmartinez24) [inline brent/index.html:5]: nice work 👍

File: brent/index.html (hunk 3 above)
Comment (wejrowski) [reply brent/index.html:6]: 👍 Nice comment Alex!

**Review by vmartinez24 — APPROVED** [review]

**Review by Koofantse — APPROVED** [review]
👍  I THINK I GOT IT

File: brent/index.html (hunk 4 above)
Comment (wejrowski) [reply brent/index.html:5]: Thanks!

**Review by adbruner — APPROVED** [review]
Ship it!

**Pull request merged**